internal/pkg/plan/persist.go

```diff
@@ -85,9 +85,12 @@ func (b *persistPlanBuilder) tryAddConfig(projectPath string, branch *model.Bran
 	// Search for config rows
 	for _, path := range b.UntrackedDirs() {
 		if rowAction := b.tryAddConfigRow(path, projectPath, configKey); rowAction != nil {
-			// Store row action inside config action too.
-			// Config ID can be then set to the rows on invoke.
-			action.Rows = append(action.Rows, rowAction)
+			// Set config ID to row key on config persist
+			rowAction := rowAction
+			action.OnPersist = append(action.OnPersist, func(parentKey model.ConfigKey) {
```

Comment:
`persist` je akcia/command, ktory najde objekty na disku, ktore nemaju priradene ID:
- priradi im ID, (vid `remote.TicketProvider`)
- a prida ich do `manifest.json`

Priklad: 
- mas v project dir, nejaky config, ... skopirujes cely adresar s configom, a zavolas `kbc persist` -> vysledne mas 2 rovnake configy s unikatnymi ID

Reply:
Cely princip je easy, prechadzaju sa vsetky `UntrackedPaths` a hlada sa ci niektora zodpoveda sablone z `Naming`:
https://github.com/keboola/keboola-as-code/blob/53f3de26d3219a493d418c5f908a75f5b8fce875/internal/pkg/plan/persist.go#L27-L34

Reply:
Ak ano, tak sa objektu priradi ID a ulozi sa do manifestu.

Reply:
Jedina komplikacia je, ked chces vytvorit `config` s `rows` v jednom kroku.
Ked sa `configu` nastavi ID, tak sa toto ID musi nastavit aj do `rows`.

Preto tu doteraz bola slice `actions.Rows`, a po nastaveni ID configu sa tieto `rows` presli a nastavilo sa im ID.

Teraz pribudol dalsi pripad a to `variables`, ... tiez mozu byt vnorene do `configu` ako `rows`.
Nechce tu pridavat dalsi field `Variables`, kedze by to bolo zbytocne vela kodu naviac.

**Namiesto toho tu menim `action.Rows`, za callbacks `action.OnPersist` a v calbacku je mozne vykonat lubovolnu operaciu.**